ironfish-rust/src/transaction/tests.rs

```diff
@@ -18,14 +18,22 @@ use ironfish_zkp::redjubjub::Signature;
 fn test_transaction() {
     let spender_key = SaplingKey::generate_key();
     let receiver_key = SaplingKey::generate_key();
+    let sender_key = SaplingKey::generate_key();
 
     // Native asset
-    let in_note = Note::new(spender_key.public_address(), 42, "", NATIVE_ASSET_GENERATOR);
+    let in_note = Note::new(
+        spender_key.public_address(),
+        42,
+        "",
+        NATIVE_ASSET_GENERATOR,
+        sender_key.public_address(),
+    );
     let out_note = Note::new(
         receiver_key.public_address(),
         40,
         "",
         NATIVE_ASSET_GENERATOR,
+        spender_key.public_address(),
```

Comment:
Should this be spender or sender?

Reply:
For the out note, this sender address is the the spender's address